src/main/java/frc/robot/end_effector/EndEffectorIOVortex.java

```diff
@@ -0,0 +1,96 @@
+package frc.robot.end_effector;
+
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkFlex;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.config.SparkBaseConfig.IdleMode;
+import com.revrobotics.spark.config.SparkFlexConfig;
+
+import au.grapplerobotics.LaserCan;
+import au.grapplerobotics.interfaces.LaserCanInterface;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+public class EndEffectorIOVortex implements EndEffectorIO {
+    
+    private final SparkFlex coralIntake = new SparkFlex(EndEffectorConstants.CORAL_INTAKE_CAN, MotorType.kBrushless);
+    private final RelativeEncoder coralIntakeEncoder = coralIntake.getEncoder();
+
+    private final SparkFlex algaeIntake = new SparkFlex(EndEffectorConstants.ALGAE_INTAKE_CAN, MotorType.kBrushless);
+    private final RelativeEncoder algaeIntakeEncoder = algaeIntake.getEncoder();
+
+    private final SparkFlex algaeWrist = new SparkFlex(EndEffectorConstants.ALGAE_WRIST_CAN, MotorType.kBrushless);
+    private final RelativeEncoder algaeWristEncoder = algaeWrist.getEncoder();
+
+    private final DigitalInput coralIntakeBeamBreakDigitalInput = new DigitalInput(0); //configure the DIO port for the Beam Break Sensor after the robot is wired.
+    private final LaserCan LaserCan = new LaserCan(0);
+
+    public EndEffectorIOVortex () {
+
+        SparkFlexConfig config = new SparkFlexConfig();
+
+        config.idleMode(IdleMode.kBrake).smartCurrentLimit(EndEffectorConstants.CORAL_INTAKE_CURRENT_LIMIT).voltageCompensation(12.0);
+        config.idleMode(IdleMode.kBrake).smartCurrentLimit(EndEffectorConstants.ALGAE_INTAKE_CURRENT_LIMIT).voltageCompensation(12.0);
+        config.idleMode(IdleMode.kBrake).smartCurrentLimit(EndEffectorConstants.ALGAE_WRIST_CURRENT_LIMIT).voltageCompensation(12.0);
+
+        config.encoder
+            .positionConversionFactor(2.0 * Math.PI)
+            .velocityConversionFactor((2.0 * Math.PI) / 60.0)
+            .uvwMeasurementPeriod(10)
+            .uvwAverageDepth(2);
+
+        coralIntake.configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
+        algaeIntake.configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
+        algaeWrist.configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
```

Comment:
You should configure the LaserCAN here.

Reply:
We don't want to record a measurement here but rather configure the LaserCAN's settings. They do this in `robotInit()` in the example snippet: https://github.com/GrappleRobotics/LaserCAN/blob/master/docs/example-java.md